### PR TITLE
Topics schema 2.0.0 issue35

### PR DESCRIPTION
**Pull request by FlatBallFlyer**
This PR addresses changes to the topic, resource, skill schemas. 

See /src/mongosh/data/enumerators.json file for explanations of the enumerated values.

### CODE REVIEW

File: src/mongosh/data/partners.json [hunk 1]
```diff
@@ -114,14 +98,10 @@
...
         "contacts": [
             {
-                "$oid": "aaaa00000000000000000019",
-                "name": "Sophia Smith",
-                "phone": "546-925-3228"
+                "$oid": "aaaa00000000000000000019"
             },
             {
-                "$oid": "aaaa00000000000000000020",
-                "name": "Mia Smith",
```
Comment (GrahamPaasch) [inline src/mongosh/data/partners.json:123]: Is it not necessary to track names and contact info? Is this a part of something else?

**Review by GrahamPaasch — COMMENTED** [review]
looks like in partners.json the "contacts" list contains objects that once contained the name and phone, but with this merge will now only contain $oid and nothing else

File: src/mongosh/data/partners.json (hunk 1 above)
Comment (michquinn) [reply src/mongosh/data/partners.json:123]: Looks like it's in the person/people data, for example:
https://github.com/agile-learning-institute/mentorHub-mongodb/blob/95a38918dd673567a55558cb408690e05cac0e20/src/mongosh/data/partners.json#L99-L106
https://github.com/agile-learning-institute/mentorHub-mongodb/blob/b5baa44b75b9d7f754c7fd5a0cf7e04990b53534/src/mongosh/data/people.json#L577-L601)

Comment (FlatBallFlyer) [reply src/mongosh/data/partners.json:123]: That is correct - if you want names and phone numbers (like the API returns) you will use a lookup() function when quering the data to fetch that from the person collection.

You can find out more about the patterns in [this article on schema best practices](https://www.mongodb.com/developer/products/mongodb/mongodb-schema-design-best-practices/) 

PS The change to the Partner schema and data was approved previously and inadvertently regressed. This was just returning those files to their previous state. This PR stated explicitly that I wanted a review of Topic, Skill, and Resource.

Comment (GrahamPaasch) [reply src/mongosh/data/partners.json:123]: Got it, approved

**Review by GrahamPaasch — APPROVED** [review]

File: src/mongosh/data/enumerators.json [hunk 2]
```diff
@@ -1,93 +1,114 @@
...
+                "Linux": "Boots to Linux (dual boot or vm)",
+                "Windows": "Uses WSL for bash/linux"
+            },
+            "cadence": {
+                "As Needed": "Meetings are not fixed",
+                "Monthly": "Meetings are once a month",
+                "Every 2 Weeks": "Meetings are once every two weeks",
+                "Weekly": "Meetings are once every week",
+                "Twice a Week": "Meetings are twice every week",
+                "Daily": "Meetings are every day"
```
Comment (michquinn) [inline src/mongosh/data/enumerators.json:36]: These are fine, though you could also consider using the ISO 8601 interval format:
- Monthly: R/P1M
- Every 2 weeks: R/P2W
- Weekly: R/P1W
- Twice a week: R/P0.5W
- Daily: R/P1D

For more info:
https://en.wikipedia.org/wiki/ISO_8601#Repeating_intervals
https://resources.data.gov/schemas/dcat-us/v1.1/iso8601_guidance/#accrualperiodicity

File: src/mongosh/data/enumerators.json [hunk 3]
```diff
@@ -1,93 +1,114 @@
...
-            "Tutorial":"",
-            "Video":"",
-            "Video Series":"",
-            "Website":""
+        "resources": {
+            "status": {
+                "Active": "An active resource",
+                "Archived": "Soft Delete Indicator"
+            },
+            "tags": {
```
Comment (michquinn) [inline src/mongosh/data/enumerators.json:95]: Consider adding a couple for accessibility purposes like "subtitles" OR "transcript" for audio content

**Review by michquinn — COMMENTED** [review]
Date values in the data files should either be in UNIX time or ISO 8601 format as per https://www.mongodb.com/docs/manual/reference/mongodb-extended-json/#mongodb-bsontype-Date

Also left 2 comments, otherwise looks good

**Pull request merged**